n4js-libs/packages/n4js-mangelhaft-cli/tests/npm-test.sh

```diff
@@ -12,8 +12,12 @@
 set -e
 cd `dirname $0`/TestPrj
 
+echo "Create test catalog"
+../../../n4js-cli/bin/n4jsc.js compile ../../../.. --testCatalog ./test-catalog.json
```

Comment:
Should we have a way to pass the path to n4jsc.js as an env variable?

Reply:
I agree and actually it is already available as `n4jsc` (like `mangelhaft-cli`), but I wanted to do it similar to the following call to mangelhaft.